microsoftgraph/microsoft-graph-explorer
# fixes broken docs link in sample query

## Overview

Changes the broken links in the samples for 'all items in the drive'

## Testing Instructions

* Click the documentation link on 'all the items in my drive'
* Confirm it goes to 'https://docs.microsoft.com/en-us/graph/api/driveitem-list-children?view=graph-rest-1.0'
